### PR TITLE
feat: config center will check plugin_metadata

### PR DESCRIPTION
**Pull request by fesily**
### Description

Fixes # (issue)

### Checklist

- [ ] I have explained the need for this PR and the problem it solves
- [ ] I have explained the changes or the new features added to this PR
- [x] I have added tests corresponding to this change
- [ ] I have updated the documentation to reflect this change
- [x] I have verified that this change is backward compatible (If not, please discuss on the [APISIX mailing list](https://github.com/apache/apisix/tree/master#community) first)

### CODE REVIEW

**Comment by fesily** [comment]
I noticed the `plugin_metadata` was only checked by admin api on etcd. It's necessary check the plugin meta schema on any config center. #7310

**Review by spacewander — DISMISSED** [review]

File: t/config-center-yaml/plugin-metadata.t [hunk 1]
```diff
@@ -33,8 +33,8 @@ _EOC_
 
     $block->set_value("yaml_config", $yaml_config);
 
-    if (!$block->no_error_log) {
-        $block->set_value("no_error_log", "[error]");
+    if (!$block->no_error_log && !$block->error_log) {
+            $block->set_value("no_error_log", "[error]");
```
Comment (spacewander) [inline t/config-center-yaml/plugin-metadata.t:37]: Sorry for my mistake. Please use a 4-spaces indentation.

Comment (fesily) [reply t/config-center-yaml/plugin-metadata.t]: The codelint is powerless

Comment (fesily) [reply t/config-center-yaml/plugin-metadata.t]: I'm not notice this.

**Review by spacewander — APPROVED** [review]

**Review by tzssangglass — APPROVED** [review]

**Pull request merged**